aws-lc-rs/tests/ecdsa_tests.rs

```diff
@@ -151,6 +153,7 @@ fn signature_ecdsa_verify_asn1_test() {
                 ("P-256", "SHA384") => &signature::ECDSA_P256_SHA384_ASN1,
                 ("P-384", "SHA256") => &signature::ECDSA_P384_SHA256_ASN1,
                 ("P-384", "SHA384") => &signature::ECDSA_P384_SHA384_ASN1,
+                // ("P-521", "SHA512") => &signature::ECDSA_P521_SHA512_ASN1,
```

Comment:
We should have some test vectors to cover `ECDSA_P521_SHA512_ASN1` as well. We could possibly re-use the "fixed" test vectors if we fix the format.

Reply:
format figured out and added tests 👍